lua/octo/writers.lua

```diff
@@ -148,10 +152,13 @@ function M.write_details(bufnr, issue, update)
   local details = {}
 
   -- author
-  local author_vt = {
-    {"Created by: ", "OctoNvimDetailsLabel"},
-    {issue.author.login, "OctoNvimDetailsValue"}
-  }
+  local author_vt = {{"Created by: ", "OctoNvimDetailsLabel"}}
+  local author_bubble = bubbles.make_user_bubble(
```

Comment:
You can decide. I mean I added them because I think it is nice. That's why it is there.
In first place it was not the intention to just bubble it, but highlight the author if it is the viewer himself. And as the colored bubble was the choice of doing so... Though I still like the general bubble. :grimacing: 
So if you don't like it, I can just remove that. Else I'll add them for the other locations as well. Please note your decision. :upside_down_face:

Reply:
Lets keep it but then, lets add it to assignees and reviewers as well for consistency and to highlight what is the viewer role in the issue/pr

Reply:
Okay, I extended it for all. And with the additional style configuration options you can "disable" it where ever you like.
One note: PR reviewers do not seem to work atm. The API always states there are no.